docs/sources/reference/components/otelcol/otelcol.receiver.awss3.md

```diff
@@ -0,0 +1,222 @@
+
+The `start_time` and `end_time` fields use one of the following time formats: RFC3339, `YYYY-MM-DD HH:MM`, or `YYYY-MM-DD`. When using `YYYY-MM-DD`, the time defaults to `00:00`.
+
+{{< admonition type="note" >}}
+Time-based configuration (`start_time` and `end_time` arguments) can't be combined together with [`sqs`][] block.
+
+[`sqs`]: #sqs
+
+{{< /admonition >}}
+
+Refer to the upstream receiver [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/{{< param "OTEL_VERSION" >}}/receiver/awss3receiver#configuration) for more details.
+
+## Blocks
+
+You can use the following blocks with `otelcol.receiver.awss3`:
+
+| Block                          | Description                                                                  | Required                                  |
+| ------------------------------ | ---------------------------------------------------------------------------- | ----------------------------------------- |
+| [`s3downloader`][s3downloader] | Configures S3 downloader.                                                    | yes                                       |
+| [`sqs`][sqs]                   | Configures SQS queue configuration for receiving object change notification. | Required if fetching by SQS notification. |
+| [`output`][output]             | Configures where to send received telemetry data.                            | yes                                       |
+
+[s3downloader]: #s3downloader
+[sqs]: #sqs
+[output]: #output
+
+### `s3downloader`
+
+{{< badge text="Required" >}}
+
+The `s3downloader` block contains AWS S3 downloader related configuration to control things like bucket, prefix, batching, connections, retries, etc.
+
+The following arguments are supported:
+
+| Name                    | Type     | Description                                                                                           | Default       | Required |
+| ----------------------- | -------- | ----------------------------------------------------------------------------------------------------- | ------------- | -------- |
+| `s3_bucket`             | `string` | S3 bucket.                                                                                            |               | yes      |
+| `s3_prefix`             | `string` | Prefix for the S3 key (root directory inside bucket).                                                 |               | yes      |
+| `endpoint_partition_id` | `string` | Partition id to use if `endpoint` is specified.                                                       | `"aws"`       | no       |
+| `endpoint`              | `string` | Overrides the endpoint used by the exporter instead of constructing it from `region` and `s3_bucket`. |               | no       |
+| `file_prefix`           | `string` | File prefix defined by user.                                                                          |               | no       |
```

Comment:
This could use a better description, what does this do? Filters the files to _only_ download the ones with the prefix?